File: liqp-core/src/main/java/liqp/tags/Raw.java
package liqp.tags;

import liqp.tag.LTag;
import liqp.node.LNode;
import liqp.context.LContext;

public class Raw extends LTag {

    /*
     * temporarily disable tag processing to avoid syntax conflicts.
     */
    @Override
    public Object render(LContext context, LNode... nodes) {
        return nodes[0].render(context);
    }
}

File: liqp-core/src/main/kotlin/liqp/tags/Raw.kt
package liqp.tags

import liqp.context.LContext
import liqp.node.LNode
import liqp.tag.LTag

class Raw : LTag() {

  /**
   * temporarily disable tag processing to avoid syntax conflicts.
   */
  override fun render(context: LContext, vararg nodes: LNode): Any? {
    return nodes[0].render(context)
  }
}
